plugins/woocommerce/client/admin/client/settings-payments/onboarding/providers/woopayments/steps/test-account/index.tsx

```diff
@@ -463,156 +426,11 @@ const TestAccountStep = () => {
 	};
 
 	if ( status === 'success' ) {
-		// Render success state.
```

Comment:
All this was removed in favor of a redirection to WC Home.

Even if it wasn't removed, we could have made it a separate sub-step to reduce this step's content (it's getting large)